$NetBSD: patch-ae,v 1.3 2009/05/17 22:28:51 dholland Exp $

Avoid invalid shell syntax.

Upstream: already fixed in upstream git.

--- configure.orig	2009-05-17 18:18:32.000000000 -0400
+++ configure	2009-05-17 18:18:45.000000000 -0400
@@ -6655,8 +6655,9 @@ fi
 
 done
 
-	for ac_header in
+	for ac_header in fnord
 do
+break;
 as_ac_Header=`echo "ac_cv_header_$ac_header" | $as_tr_sh`
 if { as_var=$as_ac_Header; eval "test \"\${$as_var+set}\" = set"; }; then
   { echo "$as_me:$LINENO: checking for $ac_header" >&5
